exchanges/poloniex/poloniex.go

```diff
@@ -785,7 +785,7 @@ func (p *Poloniex) SendAuthenticatedHTTPRequest(method, endpoint string, values
 	headers := make(map[string]string)
 	headers["Content-Type"] = "application/x-www-form-urlencoded"
 	headers["Key"] = p.API.Credentials.Key
-	values.Set("nonce", p.Requester.GetNonce(true).String())
+	values.Set("nonce", strconv.FormatInt(time.Now().UnixNano(), 10))
```

Comment:
Would you be able to elaborate on this `Fixes nonce generation issue as it requires unixnano timestamp.`?
`p.Requester.GetNonce(true)` is a unixnano timestamp.

Reply:
Doesn't like the timestamp to be incremented needs to be within a certain threshold per call. Don't have funds on the exchange, need to do a quick limit order submissions to test if we need to add in specific padding between calls now that I think about it.

Reply:
This does not seem to match their documentation:
>Additionally, all queries must include a "nonce" POST parameter. The nonce parameter is an integer which must always be greater than the previous nonce used and does not need to increase by one. Using the epoch in milliseconds is an easy choice here but be careful about time synchronization if using the same API key across multiple servers.
Did you have multiple instances running by any chance?

Reply:
![image](https://user-images.githubusercontent.com/9391715/89977827-734c7700-dcaf-11ea-9a25-e49250c3db57.png)

Tested with for loop deploying limit order. The update seems fine. Just need somebody else to confirm.

Reply:
> Would you be able to elaborate on this `Fixes nonce generation issue as it requires unixnano timestamp.`?
> `p.Requester.GetNonce(true)` is a unixnano timestamp.

GetNonce() is technically only the current time in UnixNano the first request additional requests by the same requestor instance just increment by one this has thrown exchanges off in the past as well